### ISSUE TITLE
Elmo embeddings issue

### ISSUE THREAD

**Issue by ga263** [amaiya/ktrain#108]
```
/usr/local/lib/python3.6/dist-packages/ktrain/text/ner/anago/preprocessing.py in activate_elmo(self)
     82         if not hasattr(self, 'elmo'): self.elmo=None
     83         if self.elmo is None:
---> 84             self.elmo = Elmo(options_file, weight_file, 2, dropout=0)
     85         self.use_elmo = True
     86 

NameError: name 'Elmo' is not defined
```
Running NER with any of the models that use Elmo embeddings breaks with this error. Please help and thanks for your work! :)

**Comment by amaiya** [comment]
Thanks for reporting this issue.   Becuase TF-HUB does not yet support Elmo for TF2, *ktrain* uses AllenNLP to generate Elmo embeddings.     When Elmo embeddings are selected, *ktrain* is supposed to check if AllenNLP is installed and issue an error if it is not.  There was a bug with this check.

I've fixed it in 0.12.1 so that *ktrain* instructs user to install AllenNLP if a model with Elmo embeddings is selected.

In your case, if you install AllenNLP with `pip3 install allennlp`, then the models using Elmo embeddings  (e.g., `bilstm-crf-elmo`) should work.

**Comment by ga263** [comment]
I installed ```ktrain==0.12.1``` and installed ```allennlp```. That did the trick, thanks!
